core/pom.xml

```diff
@@ -616,6 +626,36 @@
         <script.extension>.sh</script.extension>
       </properties>
     </profile>
+    <profile>
+      <id>opentelemetry</id>
+      <properties>
+        <opentelemetry.exclude></opentelemetry.exclude>
+      </properties>
+      <dependencies>
+        <dependency>
+          <groupId>io.opentelemetry</groupId>
+          <artifactId>opentelemetry-exporter-otlp</artifactId>
+          <version>1.41.0</version>
+          <exclusions>
+            <exclusion>
+              <groupId>io.opentelemetry</groupId>
+              <artifactId>opentelemetry-sdk-extension-autoconfigure-spi</artifactId>
+            </exclusion>
+          </exclusions>
+        </dependency>
+        <dependency>
+          <groupId>io.opentelemetry</groupId>
+          <artifactId>opentelemetry-sdk</artifactId>
+          <version>1.41.0</version>
+        </dependency>
+        <dependency>
+          <groupId>com.squareup.okhttp3</groupId>
+          <artifactId>okhttp</artifactId>
+          <version>3.13.0</version>
```

Comment:
Please note that Apache Spark's `okhttp` is 3.12.12. So, we had better use 3.12.12 even for the test case.

https://github.com/apache/spark/blob/9fc1e05224d2b4207885a1d4f822fed6db0cb1a1/dev/deps/spark-deps-hadoop-3-hive-2.3#L226